src/net_processing.cpp

```diff
@@ -2959,58 +2959,37 @@ bool static ProcessMessage(CNode* pfrom, const std::string& strCommand, CDataStr
     }
 
     if (strCommand == NetMsgType::PONG) {
-        int64_t pingUsecEnd = nTimeReceived;
+        // We didn't send a ping, there shouldn't be a pong
```

Comment:
Instead of adding a comment here, we could make the LogPrint more informative (which would self comment the code).